EC2Reporter/EC2Reporter.py

```diff
@@ -172,6 +187,7 @@ def main(argv=None):
     actions.add_argument("--report", dest="report", type=str, help="Submit the given textual report", metavar="TEXT")
     actions.add_argument("--report-from-file", dest="report_file", type=str, help="Submit the given file as textual report", metavar="FILE")
     actions.add_argument("--cycle", dest="cycle", type=str, help="Cycle the pool with the given ID", metavar="ID")
+    actions.add_argument("--disable", dest="disable", type=str, help="Disable the pool with the given ID", metavar="ID")
```

Comment:
If you add --disable, please also add --enable. We need both anyway for the coverage stuff if we don't want to manually kick off coverage generation.